app/models/round.rb

```diff
@@ -3,4 +3,8 @@ class Round < ApplicationRecord
   belongs_to :team
   belongs_to :movie, optional: true
   has_many :screenings, dependent: :destroy
+
+  def number_in_team
+    team.rounds.order(created_at: :asc).where('created_at < ?', created_at).count + 1
```

Comment:
This is more or less how I think we can compute it, yes. Not sure if it isn't easier and/or more performant to use the ID which is **guaranteed** to be always crescent (so I don't think it's a dirty hack).
Sure, this way looks more clear, but I think it can be dangerous to not compute it in a really fast fashion.
Just to think about.